docs/content/en/docs/reference/baremetal/bare-preparation.md

```diff
@@ -5,3 +5,71 @@ weight: 20
 description: >
   Set up a Bare Metal cluster to prepare it for EKS Anywhere
 ---
+After gathering hardware described in Bare Metal [Requirements]({{< relref "./bare-prereq" >}}), you need to prepare the hardware and create a CSV file describing that hardware.
+
+## Prepare hardware
+To prepare your computer hardware for EKS Anywhere, you need to connect your computer hardware and do some configuration.
+Once the hardware is in place, you need to:
+
+* Obtain IP addresses to assign to the IPMI and regular NIC interface on each machine in the cluster.
+* Obtain the gateway address for your network to reach the Internet.
+* Obtain the IP address for your DNS servers.
+* Make sure the following settings are in place:
+  * UEFI is enabled on all target cluster machines
+  * PXE boot is enabled on each NIC
+  * PXE is set as the first device in each machine's boot order
+  * IPMI over LAN is enabled on the IPMI interfaces
+* Go to the IPMI settings for each machine and set the IP address (bmc_ip), username (bmc_username), and password (bmc_password) to use later in the CSV file.
+
+## Prepare hardware inventory
+Create a CSV file to provide information about all physical machines that you are ready to add to your target Bare Metal cluster.
+This file will be used:
+
+* When you generate the hardware file to be included in the cluster creation process described in the [Create Bare Metal production cluster]({{< relref "../../getting-started/production-environment" >}}) Getting Started guide.
+* To provide information that is passed to each machine from the Tinkerbell DHCP server when the machine is initially PXE booted.
+
+The following is an example of an EKS Anywhere Bare Metal hardware CSV file:
+
+```
+hostname,bmc_ip,bmc_username,bmc_password,mac,ip_address,netmask,gateway,nameservers,labels,disk
+eksa-cp01,10.10.44.1,root,PrZ8W93i,CC:48:3A:00:00:01,10.10.50.2,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+eksa-cp02,10.10.44.2,root,Me9xQf93,CC:48:3A:00:00:02,10.10.50.3,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+eksa-wk01,10.10.44.3,root,Z8x2M6hl,CC:48:3A:00:00:03,10.10.50.4,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+eksa-wk02,10.10.44.4,root,B398xRTp,CC:48:3A:00:00:04,10.10.50.5,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+eksa-wk03,10.10.44.5,root,w7EenR94,CC:48:3A:00:00:05,10.10.50.6,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
```

Comment:
I would put 3 control planes and 2 worker nodes here and provide the labels
```suggestion
hostname,bmc_ip,bmc_username,bmc_password,mac,ip_address,netmask,gateway,nameservers,labels,disk
eksa-cp01,10.10.44.1,root,PrZ8W93i,CC:48:3A:00:00:01,10.10.50.2,255.255.254.0,10.10.50.1,8.8.8.8|8.8.4.4,type=cp,/dev/sda
eksa-cp02,10.10.44.2,root,Me9xQf93,CC:48:3A:00:00:02,10.10.50.3,255.255.254.0,10.10.50.1,8.8.8.8|8.8.4.4,type=cp,/dev/sda
eksa-cp03,10.10.44.3,root,Z8x2M6hl,CC:48:3A:00:00:03,10.10.50.4,255.255.254.0,10.10.50.1,8.8.8.8|8.8.4.4,type=cp,/dev/sda
eksa-wk01,10.10.44.4,root,B398xRTp,CC:48:3A:00:00:04,10.10.50.5,255.255.254.0,10.10.50.1,8.8.8.8|8.8.4.4,type=worker,/dev/sda
eksa-wk02,10.10.44.5,root,w7EenR94,CC:48:3A:00:00:05,10.10.50.6,255.255.254.0,10.10.50.1,8.8.8.8|8.8.4.4,type=worker,/dev/sda
```

Reply:
Done.